repos/fdbt-site/src/pages/api/managePassengerGroup.ts

```diff
@@ -136,6 +137,26 @@ export const collectErrors = async (userInput: GroupPassengerTypeDb, nocCode: st
             userInput: userInput.name || '',
         });
     }
+
+    // make sure underlying passenger types are not duplicated
+
+    const fullGroupPassengerType = await convertToFullPassengerType(userInput, nocCode);
+
+    const seen = new Set();
+    let typeName = '';
+
+    const hasDuplicates = fullGroupPassengerType.groupPassengerType.companions.some((item) => {
+        typeName = item.passengerType;
+        return seen.size === seen.add(item.passengerType).size;
+    });
+
+    if (hasDuplicates) {
+        errors.push({
+            errorMessage: `A group cannot contain multiple "${typeName}" passenger types`,
```

Comment:
Think you need to sentenceCase this otherwise you'll get things like youngPerson